src/panel/common/commonActions.js

```diff
@@ -54,10 +76,27 @@ function isInvalidTags(tagArr) {
 	return false;
```

Comment:
```
function addNewList(title, tagArr) {
	title = title.trim();

	if (isInvalidTitle(title) || isInvalidTags(tagArr) || isDuplicateTitle(title)) {
             indicateInvalidStuff();
             return;
	}
``` 
instead of the current placement of indicateInvalidStuff(). 
I realise that these changes were made in another pull that I overlooked ☺️